### PR TITLE
fix: progressCircle, animDict, and revert notify

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

create progressCircle properly, utilize lib.requestAnimDict, fix cache.ped placements, revert notify back to the QB event

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server.lua [hunk 1]
```diff
@@ -33,7 +33,7 @@ end
 ---@param amount integer Amount of item to be added to inventory
 local function addItem(item, amount)
     if onCooldown(20) then return end
-    local player = QBCore.Functions.GetPlayer(source)
+    local Player = QBCore.Functions.GetPlayer(source)
```
Comment (Manason) [inline server.lua]: camelCase https://manason.github.io/effective-fivem-lua/variables/#camelcase-non-constant-local-variable-names

**Review by Manason — APPROVED** [review]

File: server.lua (hunk 1 above)
Comment (mafewtm) [reply server.lua]: It was changed as it was throwing a unused variable linting error. Most of the qb serverside lua has `local Player = QBCore.Functions.GetPlayer(source)` so I was sticking with the theme.

Comment (Manason) [reply server.lua]: It's unused because the following line needs to be lower case as well.

Comment (mafewtm) [reply server.lua]: Just pushed another commit to fix it for the whole file

**Review by YvanCywan — APPROVED** [review]

**Pull request merged**